_pages/trivia.html

```diff
@@ -24,7 +24,7 @@ <h1>{{ page.heading }}</h1>
             <img
               class="thumbnail"
               src="/static/img/theme/hero-lemur.svg"
-              alt="Photo of a future trivia winner" />
+              alt="" />
```

Comment:
Wouldn't this prevent a person using a screen reader from knowing they could have their image uploaded on this page?

Reply:
We can certainly communicate that if that’s the goal! I’d recommend being more descriptive of the image then:

```suggestion
              alt="Your photo could be here as a future trivia winner. Currently a placeholder illustration with a smiling lemur on a branch" />
```